sfbrigade/datasci-earthquake
# add map and tweak responsiveness of its container

# Description

Add Mapbox map to main page and . Also adds a handful of example soft stories as [Mapbox markers](https://docs.mapbox.com/help/troubleshooting/markers-vs-layers/#markers).

Closes #72

## Type of changes

## Testing

## How to test

Run app, open index page and observe that map of SF is showing. Compare to Figma design. Attempt to pan/zoom with mouse and touch across diff devices and/or browser view port sizes.

Please also refer to requirements in #72, also linked in the right panel under "Development".
